### DISCUSSION TITLE
수도권 1호선과 통근형 도색 분리

### DISCUSSION THREAD

**Discussion by EightonEight** [KoreanGRF/KoreanTrainSet#301] [건의(Suggestions)]
수도권 1호선, 통근형 도색이 지나치게 많아 분리하는 것에 대해 의견수렴이 필요

### 현행 ###
![현행](https://user-images.githubusercontent.com/77984422/119989259-f0a86480-c001-11eb-8efb-7efc9514b054.png)

### 개선(1안) ###
![개선](https://user-images.githubusercontent.com/77984422/119981994-40cef900-bff9-11eb-9b76-a7172759bee4.png)
* **``321K_BIKE``, ``371K_381K_1ST``, ``371K_BLANK``, ``381K_2ND``, ``391K``, ``3XXK_AL``, ``3XXK_ST``를 별도 분리**
  -  ``3XXK_ST``는 #156, #230 의 사례와 같이 **Korail 31xk**와 **Korail 321k, 331k ST**로 분리하나 스프라이트는 공유 

* 도색개조가 없는 ``361K_VVVF``(경춘선)와 통합, 도색개조가 있는 수인분당선과는 분리 유지

### 개선(2안) ###
![개선2](https://user-images.githubusercontent.com/77984422/119988549-30228100-c001-11eb-94fe-dd5bdfff9b16.png)
* **1안과 동일하게 분리하나 수인분당선까지 통합**

**Comment by telk5093** [comment]
차량을 분리할 경우 이전 버전과의 호환을 끊어야 할 수도 있습니댜.
분리한다면 개인적으로는 1안을 생각 중입니다.

**Comment by opentrain** [comment]
제 의견은 기존대로 사용해도 나쁘지않는거같습니다. 여러도색의 선택지는 고를수있으니까요
만약 분리한다면 열차 도색 찾는거도 쉽지않겠지만 저 또한 1안이 괜찮다고 생각합니다.

**Comment by telk5093** [comment]
Related with #398

**Comment by EightonEight** [comment]
### 개선(3안)
![#246-5](https://github.com/KoreanGRF/KoreanTrainSet/assets/77984422/859e95d1-239b-4815-982c-553de6280059)
* **모든 차량을 개별 노선으로 분리**
  - 중복되는 스프라이트는 1안과 동일한 방식으로 공유
  - 신규: 경의중앙선, 경강선, 동해선, 서해선, 대구권 광역철도
  - 유지: 수인분당선, 경춘선, 인천공항철도, 신분당선
* **#398 을 참고하여 그룹화**
  - 그룹화 기준은 [광역전철 노선 목록](https://namu.wiki/w/%EA%B4%91%EC%97%AD%EC%A0%84%EC%B2%A0#s-2.1)을 따름

**Reply by telk5093** [reply]
Not bad

**Reply by telk5093** [reply]
* 경의중앙선: 사실상 수도권 1호선과 차량이 다르지 않으므로 분리하는 것이 오히려 코드 관리 측면에서 불편함 → 수도권 1호선으로 유지
 * 경강선: 아직 도색이 하나 뿐이라 분리할 경우 마찬가지로 코드 관리 측면에서 불편함 → 수도권 1호선으로 유지
 * 인천공항철도: 광역철도이지만 공항철도라는 위상을 감안하여 광역전철 노선으로 묶지 않음
 * 수인분당선, 경춘선, 신분당선: 묶음화
 * 서해선, 동해선: 신규 분리 후 묶음화

**Comment by telk5093** [comment]
9455f3d0c4acddeca02c55a2828c75f6c3a4c5e6 에서 구현됨
